src/main/java/io/seqera/tower/cli/responses/runs/RunViewMetrics.java

```diff
@@ -92,35 +170,60 @@ private void processDataTable(List<Map<String, Object>> metricData, PrintWriter
         table.print();
     }
 
-    @Override
-    public void toString(PrintWriter out) {
-        List<String> fields = columns.stream().map(Enum::name).collect(Collectors.toList());
-        List<String> cols = new ArrayList<>();
-        cols.add("process");
-        cols.add("metric");
-        cols.addAll(fields);
-
-        if (!metricsMem.isEmpty()) {
-            out.println(ansi(String.format("%n%n    @|bold  Memory Metrics|@%n    ----------------%n")));
-            processDataTable(metricsMem, out, cols);
-        }
-
+    /**
+     * Process data into a summarized or condensed table.
+     *
+     * @param metricData
+     * @param out
+     * @param cols
+     */
+    private void processDataReducedTable(List<Map<String, Object>> metricData, PrintWriter out, List<String> cols) {
+        TableList table = new TableList(out, cols.size(), cols.toArray(new String[0])).sortBy(0);
+        table.setPrefix("    ");
 
-        if (!metricsCpu.isEmpty()) {
-            out.println(ansi(String.format("%n%n    @|bold  CPU Metrics|@%n    ----------------%n")));
-            processDataTable(metricsCpu, out, cols);
-        }
+        metricData.forEach(processDataBlock -> {
+            processDataBlock.forEach((process, sectionDataBlock) -> {
+                List<String> cells = new ArrayList<>();
+                cells.add(process);
+                Map<String, List<String>> data = summarizeDataBlocks((Map<String, Map<String, Number>>) sectionDataBlock);
+                if (data.size() > 0) {
+
+                    // This where summarized data cells are created into a concatenated string.
+                    data.values().stream().forEach(it -> {
+                        cells.add(String.join(" ", it));
+                    });
+
+                    table.addRow(cells.toArray(new String[0]));
+                }
+            });
+        });
 
+        table.print();
+    }
 
-        if (!metricsTime.isEmpty()) {
-            out.println(ansi(String.format("%n%n    @|bold  Time Metrics|@%n    ----------------%n")));
-            processDataTable(metricsTime, out, cols);
-        }
+    /**
+     * Transform a set of data blocks into a condensed and summarized single data block.
+     *
+     * @param data
+     * @return
+     */
+    private Map<String, List<String>> summarizeDataBlocks(Map<String, Map<String, Number>> data) {
+        Map<String, List<String>> result = new HashMap<>();
+
+        data.entrySet().stream().forEach(it -> {
+            if (it.getValue() != null) {
+                Function<Number, Object> fnc = MetricFormatMapper.getMap().getOrDefault(it.getKey(), null);
+
+                for (Map.Entry<String, Number> entry : it.getValue().entrySet()) {
+                    if (!result.containsKey(entry.getKey())) {
+                        result.put(entry.getKey(), new ArrayList<>());
+                    }
 
+                    result.get(entry.getKey()).add(fnc.apply(entry.getValue()).toString());
```

Comment:
Add some padding here to align the table results:
```suggestion
                    result.get(entry.getKey()).add(String.format("%1$9s", fnc.apply(entry.getValue())));
```
Or even better, inside each formatter because then you have a better idea of the padding length.